Makefile

```diff
@@ -83,6 +83,7 @@ help: ## Display this help.
 generate: ## Generate boilerplate DeepCopy methods, manifests, and Go client
 	# Use uid and gid of current user to avoid mismatched permissions
 	set -e ; \
+	rm -rf neonvm/client neonvm/apis/neonvm/v1/zz_generated.deepcopy.go
```

Comment:
suggestion: would be nice in general have a traditional "make clean" target that drops generated and compiled things, because we already do clean ups here and there (kernel, generated files).